js/apps/system/_admin/aardvark/APP/react/build/asset-manifest.json

```diff
@@ -0,0 +1,130 @@
+{
```

Comment:
I am not sure if we want to checkin the build files anymore.
Keep them in for now until we get a decision